zeppelin-server/src/test/java/org/apache/zeppelin/rest/AbstractTestRestApi.java

```diff
@@ -232,6 +233,15 @@ protected static GetMethod httpGet(String path) throws IOException {
     return getMethod;
   }
 
+  protected static DeleteMethod httpDelete(String path) throws IOException {
+    LOG.info("Connecting to {}", url + path);
+    HttpClient httpClient = new HttpClient();
+    DeleteMethod deleteMethod = new DeleteMethod(url + path);
+    httpClient.executeMethod(deleteMethod);
```

Comment:
you will need to add the origin header:
DeleteMethod.addRequestHeader("Origin", "http://localhost");